README.md

```diff
@@ -73,7 +84,7 @@ You can look at these numbers in a few ways:
 
 - **Both `@extend` demos generate the same number of declarations as the base class, but with 55-65% more selectors.** These features, used with an exsiting class or a placeholder, generate *more* compiled CSS than any of the others.
 
-**Bottom line?** Writing CSS components with a shared base class will likely produce leaner, DRY-er compiled CSS.
+**Bottom line?** Writing CSS components with a shared base class will likely produce leaner, DRY-er compiled CSS. Making use of streamlined SCSS will keep your SCSS DRY as well.
```

Comment:
Possibly, but I think it's important to show that you can get some real DRY benefit in development (like you might by using extend, or a more verbose mixin) but still achieve a minimum in compiled code. Also, there's a typo. It should read, `Making use of a streamlined mixin will keep your SCSS DRY as well`.